plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -238,6 +239,26 @@ public PinotConfig setGrpcEnabled(boolean grpcEnabled)
         return this;
     }
 
+    public boolean isTlsEnabled()
+    {
+        return tlsEnabled;
+    }
+
+    @Config("pinot.tls.enabled")
+    public PinotConfig setTlsEnabled(boolean tlsEnabled)
+    {
+        this.tlsEnabled = tlsEnabled;
+        if (tlsEnabled && (controllerUrls != null)) {
```

Comment:
Please extract a method. You can refer to `MySqlJdbcConfig` class.

Reply:
👍